src/components/Checkbox.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <label class="checkbox">
+    <span class="text">{{ text }}</span>
+    <input
+      v-model="model"
+      type="checkbox"
+    >
+    <span class="checkmark" />
+  </label>
+</template>
+
+<script>
+export default {
+  name: 'Checkbox',
+  props: {
+    getter: { type: Boolean, required: true },
+    text: { type: String, default: '' },
+    setter: { type: Function, required: true },
+  },
+  computed: {
+    model: {
+      get() {
+        return this.getter;
+      },
+      set(newValue) {
+        this.setter(newValue);
+      },
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.checkbox {
+  position: relative;
+  padding-left: 1.3rem;
+  cursor: pointer;
+  font-size: 0.75rem;
+  user-select: none;
+}
+
+.checkbox input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.checkmark {
+  position: absolute;
+  top: 0.15rem;
+  left: 0;
+  height: 0.9rem;
+  width: 0.9rem;
+  background-color: #000;
```

Comment:
this is wrongly put into the design file, the color should be the dark grey used in Trending section for background